src/lib.rs

```diff
@@ -81,25 +81,85 @@ mod char_data;
 mod explicit;
 mod implicit;
 mod prepare;
+mod utf16;
 
 pub use crate::char_data::{BidiClass, UNICODE_VERSION};
 pub use crate::data_source::BidiDataSource;
 pub use crate::level::{Level, LTR_LEVEL, RTL_LEVEL};
 pub use crate::prepare::LevelRun;
+pub use crate::utf16::{
+    BidiInfo as BidiInfoU16, InitialInfo as InitialInfoU16, Paragraph as ParagraphU16,
```

Comment:
issue: no, I want these types to not be in the root module. Please have `pub mod utf16` instead

Reply:
Done in [74a631e](https://github.com/servo/unicode-bidi/pull/109/commits/74a631e768b9c37197d78968f3b7df0faf4681ce)